static/src/inline-svgs/icon/share-facebook.svg

```diff
@@ -0,0 +1 @@
+<svg width="32" height="32" viewBox="-2 -2 32 32"><path d="M17.9 14h-3v8h-2.9v-8h-2v-2.9h2v-2.4c0-1.9 1.1-3.7 4-3.7 1.2 0 2 .1 2 .1v3h-1.8c-1 0-1.2.5-1.2 1.3v1.8h3l-.1 2.8z"/></svg>
```

Comment:
Sure just tell me the spec of the SVG

On 28 September 2015 at 14:11, Josh Holder <notifications@github.com> wrote:

> In static/src/inline-svgs/icon/share-facebook.svg
> <https://github.com/guardian/frontend/pull/10664#discussion_r40549399>:
>
> > @@ -0,0 +1 @@
> > +<svg width="32" height="32" viewBox="-2 -2 32 32"><path d="M17.9 14h-3v8h-2.9v-8h-2v-2.9h2v-2.4c0-1.9 1.1-3.7 4-3.7 1.2 0 2 .1 2 .1v3h-1.8c-1 0-1.2.5-1.2 1.3v1.8h3l-.1 2.8z"/></svg>
>
> Since these icons are now inlined, can we remove them from the CSS to
> avoid duplication? Are there any cases where we're still using the CSS
> versions?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/10664/files#r40549399>.
>

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396